src/jsonapi.js

```diff
@@ -139,7 +139,22 @@ export const readEndpoint = (endpoint, {
         .then(json => {
           dispatch(apiRead({ endpoint, ...json }));
           onSuccess(json);
-          resolve(json);
+
+          const prevUrl = getPaginationUrl(json, 'prev', apiHost, apiPath);
+          const nextUrl = getPaginationUrl(json, 'next', apiHost, apiPath);
+
+          const getPrevPage = () => dispatch(readEndpoint(prevUrl));
+          const getNextPage = () => dispatch(readEndpoint(nextUrl));
+
+          const resObj = {
```

Comment:
I would assume insted of putting all this logics into the response, then rather have name class we could call when needed on the .then?

## Example:
```javascript
// inside the client.
import { readEndpoint, setNexturl } from 'redux-json-api';
fetchData() {
  const  { dispatch } = this.props;
  dispatch(readEndpoint('/api/v1/endpoint')
    .then(response => {
       dispatch(setNextUrl(response.data.nextUrl);
     })
}